en/releases/main.md

```diff
@@ -50,6 +50,10 @@ Please continue reading for [upgrade instructions](#upgrade-guide).
 - [SIH]: The SIH on SITL [custom takeoff location](../sim_sih/index.md#set-custom-takeoff-location) in now set using the normal unscaled GPS position values, where previously the value needed to be multiplied by 1E7.
   ([PX4-Autopilot#23363](https://github.com/PX4/PX4-Autopilot/pull/23363)).
 - [Gazebo]:
+  - Gazebo Harmonic LTS release replaces Gazebo Garden as the version supported by PX4.
```

Comment:
Yes, but perhaps not here. I will look at this next week :-)

Reply:
Decided not to mention. Why? Because I'm pushing Harmonic/Ubuntu 22.04 as the supported version. Yes, we allow 20.04, but we can't say we support it because there is no ROS 2 version on Ubuntu 22.04 that hasn't reached EOL.